modules/c++/sys/source/OSWin32.cpp

```diff
@@ -222,7 +222,7 @@ std::string sys::OSWin32::getEnv(const std::string& s) const
 
     std::vector<char> buffer(size + 1);
     DWORD retVal = GetEnvironmentVariable(s.c_str(), &buffer[0], size);
-    if (retVal != size)
+    if (retVal + 1 != size) // yay for Win32 API, see: https://msdn.microsoft.com/en-us/libary/windows/desktop/ms683188%28v=vs.85%29.aspx
```

Comment:
Make the comment a little more descriptive to explain that it's returning the number of characters stored, not including the null terminator (and still include the URL).  Move the comment above the `if` line